btcutil/psbt/psbt.go

```diff
@@ -129,7 +129,7 @@ type Packet struct {
 	Outputs []POutput
 
 	// Unknowns are the set of custom types (global only) within this PSBT.
-	Unknowns []Unknown
+	Unknowns []*Unknown
```

Comment:
Rationale behind the switch?

Reply:
Mostly to unify the three levels where unknown fields are used. That way we can have more generic handler functions for those fields in the code that requires these changes. Added the rationale to the commit message as well.